.circleci/config.yml

```diff
@@ -3,6 +3,7 @@ version: 2.1
 orbs:
   python: circleci/python@2.0.3
   ship: auth0/ship@0.5.0
+  codecov: codecov/codecov@3.1.1
```

Comment:
Maybe pin it to a major, e.g. `codecov/codecov@3`?
See https://github.com/auth0/auth0-flutter/blob/main/.circleci/config.yml#L6